src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+
+function reverseQuerySort(query) {
+  return {
+    ...query,
+    sort: _.get(query, 'sort', []).map(reverseSortDirective),
```

Comment:
Not that it matters yet, but object spreading is also a stage-3 proposal, and if we decide to [avoid code that's not in the spec](https://github.com/elastic/kibana/issues/9472), this will have to be cleaned up anyway.